src/components/ERC20Transfers/ERC20Transfers.tsx

```diff
@@ -0,0 +1,318 @@
+// import React from "react";
+// import { useMoralis } from "react-moralis";
+// import { getEllipsisTxt } from "../../helpers/formatters";
+// import { getExplorer } from "../../helpers/networks";
+import "antd/dist/antd.css";
+import { useERC20Transfers } from "../../hooks/useERC20Transfers";
+import Table from "../reusable/Table";
+// import { limitDigits } from "../../helpers/formatters";
+// import { getChainDetails } from "../../helpers/getChainDetails";
+// import { CustomImg } from "../reusable/CustomImg";
+import styled from "styled-components";
+import Blockie from "../Blockie";
+
+import React, { useState, useEffect } from "react";
+import { utils } from "ethers";
+import { fetchNftData } from "./utils";
+
+interface IStyled {
+  top?: string;
+  bottom?: string;
+  left?: string;
+  right?: string;
+  width?: string;
+}
+
+const AbsoluteImgContainer = styled("div")`
+  position: absolute;
+  top: -0.3125rem;
+  top: ${(props: IStyled) => props.top};
+  bottom: ${(props) => props.bottom};
+  left: ${(props) => props.left};
+  right: ${(props) => props.right};
+  width: ${(props) => props.width};
+`;
+
+function ERC20Transfers() {
+  const { ERC20Transfers, chainId } = useERC20Transfers();
+
+  // new implement
+  const [fetchData, setFetchData] = useState({});
+
+  const handleData = async () => {
+    const data = await fetchNftData();
+    setFetchData(data);
+  };
+
+  useEffect(() => {
+    handleData();
+  }, []);
+
+  const mockData = [
+    {
+      address: "0x263026e7e53dbfdce5ae55ade22493f828922965",
+      block_hash:
+        "0xd933d2c891c310a2b7a535080366fbe45e31d984b49ea87977cd3a1a27151b18",
+      block_number: "32492335",
+      block_timestamp: "02/08/2022 03:15PM",
+      transaction_hash:
+        "0x20150f86e4af053bf30d33e1ad63fb6c4fbe353c423786256251c6188cdbf631",
+      value: 108,
+      to_address: "0xdB75a62b27D8F2EdBa9119F741bd317139b55dcE",
+      from_address: "the-pizza-guy.eth",
+      token: {
+        address: "-0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48",
+        chainId: 137,
+        symbol: "USDC",
+        logoUri:
+          "https://storageapi.fleek.co/d1921602-c1d0-4d59-82e9-e36a2947b855-bucket/App/static/token-icons/usdc.webp",
+      },
+    },
+  ];
+
+  const columns2 = [
```

Comment:
@karangorania, please rename to just ``columns``.

Reply:
done